### PR TITLE
Fix sentences for video & clean whitespace

### PR DESCRIPTION
**Pull request by Bouh**
Fix sentences for video & clean whitespace.
Tested and works on electron and web-app.

### CODE REVIEW

File: Extensions/Video/JsExtension.js [hunk 1]
```diff
@@ -386,9 +382,9 @@ module.exports = {
       .addParameter("object", _("Video object"), "VideoObject", false)
       .addParameter("operator", _("Modification's sign"), "", false)
       .addParameter("expression", _("Opacity (0-255)"), "", false)
```
Comment (4ian) [inline Extensions/Video/JsExtension.js]: You must remove these too right?

File: Extensions/Video/JsExtension.js [hunk 2]
```diff
@@ -433,9 +429,9 @@ module.exports = {
       .addParameter("object", _("Video object"), "VideoObject", false)
       .addParameter("operator", _("Modification's sign"), "", false)
       .addParameter("expression", _("Speed "), "", false)
```
Comment (4ian) [inline Extensions/Video/JsExtension.js]: Same, should be removed?

**Comment by Bouh** [comment]
Well right, just how are define the description like `Opacity (0-255)` because indeed when i delete the lines in previous commit, the description is stay present.

**Comment by 4ian** [comment]
These are added automatically when calling UseStandardOperatorParameters/UseStandardRelationalOperatorParameters: 
https://github.com/4ian/GDevelop/blob/master/Core/GDCore/Extensions/Metadata/InstructionMetadata.cpp#L91

**Comment by 4ian** [comment]
Looks good to go now!

**Pull request merged**